code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -7,8 +7,8 @@
 	icon_state = "ash_walker_nest"
 	icon_living = "ash_walker_nest"
 	faction = list("ashwalker")
-	health = 200
-	maxHealth = 200
+	health = 400
+	maxHealth = 400
```

Comment:
You can't, if you're out exploring mining.

Again, play it before you pass judgement. Or just say if you're not going to and I'll stop responding to you.